### PR TITLE
#git-4 Support line ending configuration for checkout

### PR DESCRIPTION
**Pull request by bilaltk90**

### CODE REVIEW

File: src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java [hunk 1]
```diff
@@ -48,7 +51,13 @@ public static void cleanupDotContent(String root, String[] contentProperties) th
 						lines.add(line);
 					}
 				}
-				FileUtils.writeLines(file, CharEncoding.UTF_8, lines);
+				String line = null;
```
Comment (zietas) [inline src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java]: looks like a good candidate for a private function i.e. 'getLineEnding' or something like that

File: src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java [hunk 2]
```diff
@@ -17,21 +17,24 @@
 public final class CleanHelper {
 
 	private static final Logger LOG = LoggerFactory.getLogger(CleanHelper.class);
+	private static final String UNIX = "unix";
```
Comment (zietas) [inline src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java:21]: I think the Cognifide standard is to include a blank line after field  so it's a bit easier to read it.

File: src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java [hunk 3]
```diff
@@ -51,19 +55,23 @@ public static void cleanupDotContent(String root, String lineEnding, String[] co
...
 					}
 				}
-				String line = null;
-				if (StringUtils.equals(lineEnding, UNIX)) {
-					line = UNIX_LINE_ENDING;
-				} else if (StringUtils.equals(lineEnding, DOS)) {
-					line = DOS_LINE_ENDING;
-				}
-				FileUtils.writeLines(file, CharEncoding.UTF_8, lines, line);
+				FileUtils.writeLines(file, CharEncoding.UTF_8, lines, getLineEnding(lineEnding));
```
Comment (zietas) [inline src/main/java/com/cognifide/maven/plugins/vltco/goals/clean/CleanHelper.java]: one thing is a bit confusing here

getLineEnding(lineEnding) - sounds funny if you read it outloud. 

maybe we should change the method name into something like getLineEndinchChar?

**Comment by devzbysiu** [comment]
This project is going to be archived. Closing

**Pull request closed without merging**